README.md

```
@@ -1,40 +1,50 @@
+
+You can also specify an output directory with `python -m src.cli.main path\to\datadir path\to\outputdir` (windows). Without specifying an output directory, the transformer will create one in your root directory or add the files to `output` if it already exists.
+
+
+## BRIEF PROCESS EXPLANATION: 
 
-## Notes: 
+We create a collection of each of the HSDS files by going through each input file and doing the following:
 
-We parse the input file into the form:
+parse the input file into a dictionary such as:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)